python/ray/air/train/integrations/huggingface/huggingface_trainer.py

```diff
@@ -308,20 +301,11 @@ def _validate_trainer_init_per_worker(
             )
 
     def _validate_attributes(self):
-        # exceptions first
-        if TRAIN_DATASET_KEY not in self.datasets:
-            raise KeyError(
-                f"'{TRAIN_DATASET_KEY}' key must be preset in `datasets`. "
-                f"Got {list(self.datasets.keys())}"
-            )
-        if not all(
-            key in (TRAIN_DATASET_KEY, EVALUATION_DATASET_KEY) for key in self.datasets
-        ):
-            raise KeyError(
-                f"Only '{TRAIN_DATASET_KEY}' and '{EVALUATION_DATASET_KEY}' "
-                "keys can be preset in `datasets`. "
-                f"Got {list(self.datasets.keys())}"
-            )
```

Comment:
All this is auto validated now based on dataset_config.

Reply:
Nice!